### PR TITLE
Proposal of simplification of the main README.md documentation

### PR DESCRIPTION
**Pull request by gre**
This is a proposal of documentation changes in the main README.md of the project.

The documentation is really well done and have a lot of information, but this suggest some "moves" in that documentation to try to be more welcoming of newcomers and people coming on developing the projects:

- in that sense, I suggest to put the intro to the pnpm workspace in the About intro and then folowed by the Contributing section.
- there were a bit too much "DX" centric information on the central README.md, everything on the "Tools / Scoping / Alias" which explain turborepo, filtering etc.. is not essential, Day One, for the normal dev as long as we have the _generic commands in place_. **=> if we are ok with this idea, I would then simply suggest we create a wiki page to save what we had before** It's some sort of "Advanced tooling" notes.
- Some of the most important and _generic commands in place_ however are not present in that main README. It actually took me quite a bit to understand how to run LLM in dev mode for instance. I tried to put it back in a minimal version of the Usage section. and yes, people should then open the README of each project, which is why I put it just after the Usage section and make the "apps" `open` by default so it's easier to not miss it.

TLDR: Generally speaking, I would like any dev to come in our main README and read it all without the extra detail i don't need on Day 1. before this changes, i think it is easy to get lost and disconnect at the section "Tools", missing the important rest of the doc, notably the "contributing". it's good to know the gist of how i can survive at running the project & contributing it, in the main readme.

### CODE REVIEW

**Comment by gre** [comment]
So far, I never had to use `--filter` when developing a feature. I like this documentation to exist, but it could be just linked or yes maybe put at the end.
 
ultimately it's part of turborepo documentation which we already introduce by `We use [pnpm workspaces](https://pnpm.io/) and [turborepo](https://turborepo.org/) under the hood to handle local and external dependencies, orchestrate tasks and perform various optimizations like package hoisting or [remote caching](https://turborepo.org/docs/features/remote-caching).`

**Comment by gre** [comment]
if you want, i can just put it all back but just change the ordering and add the small changes done on the "Usage" part which was to me the most important idea of this changes. @valpinkman

**Comment by valpinkman** [comment]
@gre 

> if you want, i can just put it all back but just change the ordering and add the small changes done on the "Usage" part which was to me the most important idea of this changes. @valpinkman

@gre yes please 🙏🏻  that'd be perfect

**Comment by gre** [comment]
I'll reopen a PR tomorrow. it will be easier
